packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -38,7 +38,7 @@ typedef DragTargetAcceptWithDetails<T> = void Function(DragTargetDetails<T> deta
 /// this [DragTarget] and that will not be accepted by the [DragTarget].
 ///
 /// Used by [DragTarget.builder].
-typedef DragTargetBuilder<T> = Widget Function(BuildContext context, List<T?> candidateData, List<dynamic> rejectedData);
+typedef DragTargetBuilder<T> = Widget Function(BuildContext context, List<T> candidateData, List<dynamic> rejectedData);
```

Comment:
Same here, this would break customers by requiring them to change the function signature and then they would need to update their implementations to handle when candidateData is null.